src/aio.rs

```diff
@@ -0,0 +1,265 @@
+// Copyright (C) 2019 Alibaba Cloud Computing. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Safe wrapper for Linux AIO API.
+
+#![allow(non_camel_case_types)]
+
+/* automatically generated by rust-bindgen and then manually edited */
+
+use std::io::{Error, Result};
+use std::os::raw::{c_int, c_long, c_uint, c_ulong};
+use std::ptr::null_mut;
+
+type __s16 = ::std::os::raw::c_short;
+type __u16 = ::std::os::raw::c_ushort;
+type __u32 = ::std::os::raw::c_uint;
+type __s64 = ::std::os::raw::c_longlong;
+type __u64 = ::std::os::raw::c_ulonglong;
+
+pub const IOCB_CMD_PREAD: u32 = 0;
```

Comment:
It's weird that the tests pass even though these pub constants are not documented. Any chance we are missing an `allow(missing_docs)` somewhere?

Reply:
You are right, there's no "deny(missing_docs)" in lib.rs.